dom.bs

```diff
@@ -5640,7 +5641,7 @@ The
 method, when invoked, must run these steps:
 
 <ol>
- <li><p>Let <var>doc</var> be a new <a>document</a> that is an <a>HTML document</a>.
+ <li><p>Let <var>doc</var> be a new {{HTMLDocument}}.
```

Comment:
HTMLDocument object

Reply:
This&nbsp;should&nbsp;probably be&nbsp;fixed for&nbsp;`{{XMLDocument}}`&nbsp;above as&nbsp;well.